wandb/sdk/wandb_settings.py

```diff
@@ -549,12 +558,15 @@ class Settings(BaseModel, validate_assignment=True):
     Please use `http_proxy` and `https_proxy` instead.
     """
 
-    x_runqueue_item_id: str | None = None
-    """ID of the Launch run queue item being processed."""
-
     x_require_legacy_service: bool = False
     """Force the use of legacy wandb service."""
 
+    x_run_stop_polling_interval: int = 15
```

Comment:
i think this is a really bad idea... backend already say we spam them with stop requests

Reply:
this is for my testing, i didn't want to wait haha